docs/guide/usage/integrations/home_assistant.md

```diff
@@ -248,18 +248,36 @@ mqtt:
       state_topic: "zigbee2mqtt/bridge/state"
       value_template: "{{ value_json.state }}"
       icon: mdi:router-wireless
+      entity_category: diagnostic
+      device:
+        identifiers: zigbee2mqtt
+        name: "Zigbee2MQTT"
+        model: "Zigbee2MQTT"
+        manufacturer: "Koenkk"  
```

Comment:
Lol 😄 

Please change this to `manufacturer: "Zigbee2MQTT"`, many people are responsible for creating this 😄

Reply:
Updated manufacture :)